Week2/js-exercises/ex4-collectiveAge.js

```diff
@@ -10,24 +10,27 @@
 
 function collectiveAge(people) {
   // return the sum of age for all the people
+  return people
+    .map(person => person.age)
+    .reduce((a, b) => a + b, 0);
```

Comment:
This is correct but you don't need a `map` necessarily. `b` is the full object so you can do:

`people.reduce((a, person) => a + person.age, 0)`